### ISSUE TITLE
Failed to build APK with embedded wasmedge

### ISSUE THREAD

**Issue by picolov** [WasmEdge/WasmEdge#2003]
## Description
i tried to follow this steps (https://wasmedge.org/book/en/contribute/build_from_src/android/apk.html) to build an apk that have an embedded wasmedge. 

but when building the apk always getting this problem saying it doesn't found libwasmedge.so as in the screenshot, 

i am using mac, checking on the folder it does only have libwasmedge_lib.so not libwasmedge.so, i tried to rename it but this seems to be generated so it overriden again, i search the text and find there is some kind of linking or build that might related to this libwasmedge.so <=> libwasmedge_lib.so, but i cannot grasp how this likely works to know what likely go wrong, i'm quite foreign on how cmake, build.ninja etc works. do see a pr related to a changes on august 2022 related to name change in this kind of so name, but don't know if it relate or not

### Current State

build process failed at Task :lib:buildCMakeRelWithDebInfo FAILED with error

```
Execution failed for task ':lib:buildCMakeRelWithDebInfo'.
> Expected output file at /Users/picolov/Documents/workspace/fun/WasmEdge/utils/android/app/lib/build/intermediates/cxx/RelWithDebInfo/3a4g6b6z/obj/arm64-v8a/libwasmedge.so for target wasmedge_shared but there was none
```

### Expected

build process success, and created apk artifact

## Environment

- Hardware Architecture: x86\_64
- Operating system: Mac Os Monterey 12.6

The following information is optional. Please provide them only if you have built from source.
- C++ Compiler version: ?
- CMake version: 3.18.1
- CMake flags: (e.g. `-DCMAKE_BUILD_TYPE=Release -DWASMEDGE_BUILD_AOT_RUNTIME=ON`) ?
- Boost version: ?

## Steps to Reproduce

checkout project from https://github.com/WasmEdge/WasmEdge.git , open folder /utils/android/app on Android Studio 2021.3.1 Patch 1, go to menu bar Build > Generate Signed Bundle / APK, pick generate as APK, use keystore, select release, finish

**Comment by alabulei1** [comment]
@O3Ol  @hangedfish  Could you help take a look?

**Comment by hangedfish** [comment]
Try this RP #2013

**Comment by picolov** [comment]
tested on mine, it is fixed using [#2013](https://github.com/WasmEdge/WasmEdge/pull/2013) , can build and run
